packages/gestalt/src/index.js

```diff
@@ -47,6 +47,8 @@ import TextField from './TextField.js';
 import Toast from './Toast.js';
 import Tooltip from './Tooltip.js';
 import Video from './Video.js';
+// eslint-disable-next-line import/extensions
```

Comment:
@simkessy please remove this `eslint-disable-next-line` and add `.js` the the end:

```js
import Typeahead from './Typeahead.js';
```